README.md

```diff
@@ -52,6 +52,7 @@ Currently, there are these translations of **wtfjs**:
   - [true is false](#true-is-false)
   - [baNaNa](#banana)
   - [`NaN` is not a `NaN`](#nan-is-not-a-nan)
+  - [`Object.is()` and `===` weird cases](#Object.<i></i>is()-and-===-weird-cases)
```

Comment:
Wrong link.

```suggestion
  - [`Object.is()` and `===` weird cases](#objectis-and--weird-cases)
```

Reply:
Oops my bad, thanks for catching that!

Reply:
You shouldn't edit the table of content. It has to be updated by running `npm run toc`